scanners/port_scanner.py

```diff
@@ -1,22 +1,29 @@
 import nmap
-from util.s3_helper import send_to_s3
+from lib.s3_helper import send_to_s3
 
 
 class PortScanner():
 
-    def __init__(self, hostname):
-        self._host = hostname
+    def __init__(self, hostname, arguments="-v -Pn -sT -sV --script=banner --top-ports 1000 --open -T4 --system-dns"):
```

Comment:
I don't believe users have control over the flags here, so this is not an issue.   However, if we ever expose this externally, we'd need to refactor this to avoid command injection in Lambda.

Reply:
Good point. I will add a protection against this.